### DISCUSSION TITLE
WooCommerce

### DISCUSSION THREAD

**Discussion by dawidurbanski** [wpappointments/wpappointments#83] [Ideas]
Make it possible to sell appointments via WooCommerce

**Comment by tomrobak** [comment]
+1 must have feature

**Reply by wppoland** [reply]
Thanks for the +1! WooCommerce support is confirmed as an optional integration - enable it if you need order and payment flows, leave it off to keep the site lean. Polish payment gateways via WooCommerce are tracked separately in discussion #76.
